modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/Main.scala

```diff
@@ -67,6 +68,23 @@ object Main {
       val acl                  = Acls[Future](aggregate, clock)
       val downStreamAuthClient = DownstreamAuthClient(appConfig.oidc, cl, uicl)
 
+      if (appConfig.auth.adminGroups.nonEmpty) {
+        val adminGroup = GroupRef(appConfig.oidc.issuer, appConfig.auth.adminGroups.head)
```

Comment:
shouldn't we do that for all the groups, not only for the first group?
or if we want to have only one admin group, can we change the config to a single group?
otherwise it's quite confusing.